widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.java

```diff
@@ -58,32 +58,69 @@ public AppBarView(@NonNull Context context, @Nullable AttributeSet attrs, int de
                 Utils.getTypedArrayIntegerValue(typedArray,
                         context,
                         R.styleable.AppBarView_android_backgroundTint,
-                        R.attr.gliaBrandPrimaryColor));
-        int lightColor = ContextCompat.getColor(
+                        R.attr.gliaBrandPrimaryColor
+                )
+        );
+
+        int titleTintColor = ContextCompat.getColor(
```

Comment:
this method is getting very long. can it be broken down? like `setTitleTextColor()`